packages/app/src/cli/services/build/extension.ts

```diff
@@ -141,11 +142,18 @@ export async function buildFunctionExtension(
   }
 
   try {
+    const bundlePath = extension.outputPath
+    extension.outputPath = joinPath(extension.directory, joinPath('dist', 'function.wasm'))
     if (extension.isJavaScript) {
       await runCommandOrBuildJSFunction(extension, options)
     } else {
       await buildOtherFunction(extension, options)
     }
+    if (fileExistsSync(extension.outputPath) && bundlePath !== extension.outputPath) {
+      const base64Contents = await readFile(extension.outputPath, {encoding: 'base64'})
+      await touchFile(bundlePath)
+      await writeFile(bundlePath, base64Contents)
```

Comment:
Could we use `copyFile` here instead of reading and writing?

Reply:
I did have `copyFile` here before but I need it to be in `base64` format inside the bundlePath because code in core expects this to be base64 formatted, so `copyFIle` which copies file as is doesn't work here.